src/frontend/src/lib/components/hero/HeroSignIn.svelte

```diff
@@ -1,15 +1,45 @@
 <script lang="ts">
+	import type { ComponentType } from 'svelte';
+	import IconGlasses from '$lib/components/icons/IconGlasses.svelte';
+	import IconInfinity from '$lib/components/icons/IconInfinity.svelte';
+	import IconShieldAlert from '$lib/components/icons/IconShieldAlert.svelte';
 	import ButtonAuthenticate from '$lib/components/ui/ButtonAuthenticate.svelte';
 	import { signIn } from '$lib/services/auth.services';
 	import { i18n } from '$lib/stores/i18n.store';
+
+	const infoList: { label: string; icon: ComponentType }[] = [
+		{
+			label: $i18n.auth.text.safe_access,
+			icon: IconGlasses
+		},
+		{
+			label: $i18n.auth.text.privacy_and_security,
+			icon: IconShieldAlert
+		},
+		{
+			label: $i18n.auth.text.powered_by_chain_fusion,
+			icon: IconInfinity
+		}
+	];
 </script>
 
 <div class="mb-7 mt-5 pt-2">
-	<h1 class="text-4xl">
+	<h1 class="md:text-5xl md:leading-tight">
 		{$i18n.auth.text.title_part_1}<br /><span class="text-primary"
 			>{$i18n.auth.text.title_part_2}</span
 		>
 	</h1>
 </div>
 
+<div class="mb-7 flex flex-col items-center gap-0 md:items-start md:gap-3 md:text-lg">
+	{#each infoList as { label, icon }}
+		<div class="flex items-center gap-4">
+			<div class="hidden md:block">
```

Comment:
It's correct that only the icon is hidden on mobile and not the all block? 

Unsolicited feedback: Based on the screenshot, I have to say that I find the block hard to read on mobile.

Reply:
I had both version and it seemed not so good with the icons, because the were all unaligned... but for the readability, i can try to increase the gap to make it so it is a list. See below, WDYT?


<img width="395" alt="Screenshot 2024-10-04 at 08 29 29" src="https://github.com/user-attachments/assets/89075787-34c0-4db3-9905-1c8d4ea0aced">
<img width="418" alt="Screenshot 2024-10-04 at 08 29 20" src="https://github.com/user-attachments/assets/c49d489c-5b6e-4de5-8980-b9304979fd45">

Reply:
Better yes. Personally I would probably not display this block of text at all on mobile but, I'm not a designer. Thanks.